packages/varlet-ui/src/input/Input.vue

```diff
@@ -269,6 +269,10 @@ export default defineComponent({
 
     call(bindForm, inputProvider)
 
+    onMounted(() => {
+      if (props.autofocus) focus()
+    })
+
```

Comment:
This logic may not work as expected, When use use input nested with modal component. It can't work. I think expose focus func is enough. In different scenarios user can use $refs.focus to resolve question.

Reply:
If you think props `autofocus` is necessary. My suggestion is to put it in to the input component as a parameter. Just look lie

```js

<Input :autofocus="autofocus" />

```

Reply:
By the way. If use props control Input component autoFocus behavior also can't work normal work with modal component. user still needs to perform the focus manually. So my option is `autoFocus` is no necessary!